modules/user/user.service.js

```diff
@@ -311,18 +310,17 @@ class UserService {
       });
 
       const savedUser = await user.save();
-      const token = await generateToken(savedUser._id, savedUser.email, {
-        EXPIRES_IN: undefined,
-      });
+      const token = await generateToken(savedUser._id, savedUser.email);
 
       const message = {
         from: process.env.MAIL_USER,
         to: savedUser.email,
         subject: '[Horondi] Invitation to become an admin',
-        html: SpecialUserConfirmationMessage(token),
+        html: adminConfirmationMessage(token),
       };
   
       if (process.env.NODE_ENV === 'test') {
+        console.log(token);
```

Comment:
Please, remove this useless `console.log`

Reply:
fixed